openghg/store/base/_base.py

```diff
@@ -160,6 +160,24 @@ def check_hashes(self, filepaths: multiPathType, force: bool) -> Tuple[Dict[str,
 
         return seen, unseen
 
+    def add_metadata_keys(self, data: Dict, additional_metadata: dict) -> dict:
```

Comment:
Maybe call this something else since we're updated the metadata dictionary, not just the keys. Maybe just `update_metadata`?

Reply:
Should be done